code/modules/mob/living/silicon/robot/emote.dm

```diff
@@ -76,19 +73,19 @@
 			message = "<B>[src]</B> twitches violently."
 			m_type = 1
 
-		if ("twitch_s")
+		if ("twitch_s","twitches")

```

Comment:
Ah, sorry, wasn't showing up in here.

Also, WTF? did they really need twitch to be twitches violently and twitches to just be twitches???